recipes/gtk4/meta.yaml

```diff
@@ -0,0 +1,137 @@
+{% set name = "gtk" %}
+{% set version = "4.0.3" %}
+{% set version_majmin = '.'.join(version.split('.', 2)[:2]) %}
+
+package:
+  name: gtk4  # note customization
+  version: {{ version }}
+
+source:
+  url: https://download.gnome.org/sources/{{ name }}/{{ version_majmin }}/{{ name }}-{{ version }}.tar.xz
+  sha256: d7c9893725790b50bd9a3bb278856d9d543b44b6b9b951d7b60e7bdecc131890
+  patches:
+    # https://gitlab.gnome.org/GNOME/gtk/-/merge_requests/3342
+    - gdkmacosglcontext.c.patch  # [osx]
+
+build:
+  number: 0
+  # workaround for bug in LIEF (https://github.com/lief-project/LIEF/issues/239)
+  # that resulted in missing symbols
```

Comment:
I believe this comment about LIEF should go with the following line of YAML:

```
rpaths_patcher: patchelf  # [linux]
```

If this item is intentionally being removed for this recipe, the comment should be removed.

Reply:
I removed the comment. Thanks for bringing it to my attention!